Intersect.Client.Core/MonoGame/Input/MonoInput.cs

```diff
@@ -249,6 +252,30 @@ private void CheckMouseScrollWheel(int scrlVValue, int scrlHValue)
                 new GwenInputMessage(IntersectInput.InputEvent.MouseScroll, p, MouseButton.Middle, Keys.Alt)
             );
 
+            var deltaV = Math.Sign(mMouseVScroll - scrlVValue);
+
+            var now = DateTime.Now;
+            if ((now - _lastZoomTime).TotalMilliseconds >= ZoomDelayInMilliseconds)
+
+            {
+
+                if ((InputHandler.HoveredControl == null
+                     || !InputHandler.HoveredControl.IsVisibleInTree
+                     || (InputHandler.HoveredControl is Canvas canvas && canvas == canvas.Canvas))
+                    && Globals.GameState == GameStates.InGame
+                    && Globals.Database.EnableScrollingWorldZoom)
+                {
+                if (deltaV < 0)
+                {
+                    Core.Input.HandleZoomIn(false);
+                }
+                else
+                {
+                    Core.Input.HandleZoomOut(false);
+                }
```

Comment:
Are these lines missing an indent or is this just the GitHub UI displaying it incorrectly?